onnxruntime/core/graph/graph.cc

```diff
@@ -467,6 +480,188 @@ void Node::ToProto(NodeProto& proto, bool update_subgraphs) const {
   }
 }
 
+Status Node::SaveToOrtFormat(flatbuffers::FlatBufferBuilder& builder,
+                             flatbuffers::Offset<fbs::Node>& fbs_node) const {
+  // if type is Primitive it's an ONNX function and currently we have kernel implementations for all those
+  if (func_body_ != nullptr && node_type_ != Type::Primitive) {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "Serialization of fused function body is not currently supported");
+  }
```

Comment:
nit: include some info about the node. op type, since_version and name would be good.